paddle/majel/README.md

```diff
@@ -93,6 +93,19 @@ typedef boost::variant<
 
 Because `variant` may be thought of as "multi-type, single value", we can utilize it to implement unified interfaces for PaddlePaddle.
 
+`DDim` plays two kinds of roles in Majel. First, it is used to indicate the size of a tensor. For example, we can construct a new `DArray` by following way:
```

Comment:
Should we also explain the difference betwen DDim and Dim, as well as that between DArray and Array?